<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStorage</class>
 <widget class="QWidget" name="AddStorage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="AdressBuildingLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>101</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Вулиця</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Square">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AdressStreetLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>71</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Вулиця</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdressBuilding">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddStorageButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdStorage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>232</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SquareLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>131</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Площа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdressCity">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdressStreet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IdStorageLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>11</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID складу</string>
   </property>
  </widget>
  <widget class="QLabel" name="AdressCityLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>41</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Місто</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
